.prettierrc.yaml

```diff
@@ -2,7 +2,7 @@ arrowParens: avoid
 bracketSpacing: false
 jsxBracketSameLine: false
 printWidth: 80
-proseWrap: never
+proseWrap: "preserve"
```

Comment:
Based on recent experience in other projects, I think this is the better default setting.

Reply:
Yes, why not.. no need for quotes in `yaml`